### ISSUE TITLE
Adopt towncrier or similar for generating changelogs

### ISSUE THREAD

**Issue by madig** [googlefonts/fontbakery#2191]
Just a suggestion: https://github.com/hawkowl/towncrier is a tool that uses changelog snippets people write alongside their pull requests to compile a changelog for release. Might make collaboration easier. It is used e.g. in https://github.com/pytest-dev/pytest, see https://github.com/pytest-dev/pytest/tree/master/changelog.

**Comment by davelab6** [comment]
Thanks! I think this is a good suggestion as automating the release process is a good thing if Felipe is going to do it weekly :)

**Comment by felipesanches** [comment]
yes and no. I am a fan of doing by hand for a while before automating. So that we grasp what actually needs automation.

I have a very fontbakery specific style for the changelogs, so I think simply using a generic tool could degrade the quality for a price of only an extra couple minutes per commit, perhaps.

**Comment by davelab6** [comment]
https://github.com/symless/github-query/blob/master/ghq.py also seems useful
